tracr/io/dicom/read.py

```diff
@@ -0,0 +1,85 @@
+#!/usr/bin/env python
+"""
+A script that converts dicom files to an intensity voxel array
+The resulting array is 1000 x 1024 in cross section direction and 1014 "tall"
+"""
+
+import sys, os, re
+import numpy as np
+import dicom
+
+
+def _regularize_dcm_files(dcm_files):
+	# Are we reading a file, a list of files, or files from a directory?
+	# 1. Is dcm_files a string?
+	if isinstance(dcm_files, str):
+		# yes: either a single file or a directory
+		if os.path.isdir(dcm_files):
+			# create a list of DICOM files from the contents of the directory
+			# dcm_files = dcm_files.rstrip('/')
+			rval = ['{}/{}'.format(dcm_files, fname)
+					for fname in os.listdir(dcm_files)
+					if re.match(r'.*\.dcm', fname)]
+		else:
+			# it is it's own file
+			rval = [dcm_files]
+	else:
+		# We receive a list of string filenames, what `read` expects
+		rval = dcm_files
```

Comment:
Ensure this is a list, e.g. `rval = list(dcm_files)`. This guarantees a common return type (`list`) instead of list or tuple or ndarray or whatever.